src/topojsonm/dune

```
@@ -1,4 +1,4 @@
 (library
  (name topojsone)
  (public_name topojsone)
- (libraries topojson geojsone))
+ (libraries topojson eio geojsone))
```

```suggestion
 (libraries topojson geojsone))
```

There should be no dependency on Eio for the library.